enterprise/backend/src/metabase_enterprise/serialization/v2/utils/yaml.clj

```diff
@@ -0,0 +1,43 @@
+(ns metabase-enterprise.serialization.v2.utils.yaml
+  (:require [clojure.java.io :as io])
+  (:import java.io.File
+           java.nio.file.Path))
+
+(defn- clean-string [s]
+  (if (> (count s) 100)
+      (subs s 0 100)
+      s))
```

Comment:
not totally clear from the code why this function is necessary/how it's used. it's just for truncating the labels to a reasonable length?

maybe you could make the 100 char limit a separate `def` with some kind of semantically significant name?

Reply:
Changed `clean-string` to `truncate-label` and made the 100 a `max-label-length` private def.